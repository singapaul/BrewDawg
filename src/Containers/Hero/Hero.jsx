import React from "react";
import "./Hero.scss";

const Hero = ({hideHero}) => {
  return (
    <div className="heroContainer">
      <h1>Let's grab a Brew, Dawg</h1>
      <button onClick={hideHero} className="heroContainer__button">
        <a href="#filter">Pour me a beer</a>
      </button>
    </div>
  );
};

export default Hero;
