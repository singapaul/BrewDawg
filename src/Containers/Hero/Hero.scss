@use "../../assets/_variables.scss" as *;

* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

.heroContainer {
  overflow-x: hidden;
  margin: 0;
  overflow: hidden;
  display: flex;
  background-image: url(../../assets/images/SiteLanding.png);
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  height: 100vh;
  width: 100%;
  font-size: 4rem;
  color: $color-white;
  text-align: center;
  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
  flex-direction: column;
  justify-content: space-around;

  button {
    background: transparent;
    border: 4px solid white;
    border-radius: 25px;
    padding: 15px;
    width: 30%;
    align-self: center;

    a {
      text-decoration: none;
      color: white;
      font-size: 2em;
      height: auto;
      width: auto;
    }
  }
}
