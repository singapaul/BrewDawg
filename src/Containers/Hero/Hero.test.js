import { getElementError, render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import Hero from "./Hero";

// render returns object with a container
it("should match form component snapshot", () => {
  const { container } = render(<Hero />);
  expect(container).toMatchSnapshot();
});
