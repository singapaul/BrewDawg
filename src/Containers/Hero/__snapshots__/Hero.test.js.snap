// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match form component snapshot 1`] = `
<div>
  <div
    class="heroContainer"
  >
    <img
      alt=""
      class="heroContainer__image"
      src="SiteLanding.png"
    />
    <h1
      class="heroContainer__greeting"
    >
      Let's grab a Brew, Dawg
    </h1>
  </div>
</div>
`;
